app/javascript/dashboard/App.vue

```diff
@@ -98,7 +98,7 @@ export default {
       mql.onchange = e => setColorTheme(e.matches);
     },
     setLocale(locale) {
-      this.$root.$i18n.locale = locale;
+      this.$root.$i18n.locale = locale.replace('-', '_');
```

Comment:
This may not work. With Vue 3, the `$root` element is not present. Just updating the `this.$i18n.locale` should work. Can you please test that @KokeroO